dep-sandbox/src/main/java/com/wso2telco/services/dep/sandbox/servicefactory/wallet/ListTransactionRequestHandler.java

```diff
@@ -74,10 +74,11 @@ protected boolean validate(ListTransactionRequestWrapper wrapperDTO) throws Exce
 			Validation.checkRequestParams(validationRules);
 
 		} catch (CustomException ex) {
-			LOG.error("###WALLET### Error in Validation : " , ex);
-			responseWrapper.setRequestError(constructRequestError(SERVICEEXCEPTION, ex.getErrcode(), ex.getErrmsg(),
-					wrapperDTO.getEndUserId()));
+			LOG.error("###WALLET### Error in Validations. ", ex);
+			responseWrapper.setRequestError(
+					constructRequestError(SERVICEEXCEPTION, ex.getErrcode(), ex.getErrmsg(), ex.getErrvar()[0]));
```

Comment:
just add only request error message here. set response code commonly in process() to avoid duplication work for errors